stable/logdna-agent/templates/secrets.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "name" . }}
+    chart: {{ template "chart" . }}
+    release: {{ .Release.Name }}
+type: Opaque
+data:
+  logdna-agent-key: {{ default "" .Values.logdna.key | b64enc | quote }}
```

Comment:
Don't use `default` function. Put defaults in `values.yaml`.

Reply:
We require a `key` (customer ingestion key) to be set before the chart can be installed. This approach is used by several other charts in their secrets.yaml. We were hoping this'll be okay here as well. Though if there's a different way to doing this, we can modify this as well.